### PR TITLE
Ivysaur 88 create a login page and story the authentication token

### PR DESCRIPTION
**Pull request by uzagmo**

### CODE REVIEW

**Review by Jianriu — APPROVED** [review]

File: client/src/page/loginPage.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    gap: 30px;
+
+    background-color: rgb(152, 180, 255);
+    padding-top: 30px;
+    padding-bottom: 30px;
+}
+
+.page-header{
+
+    font-family: fantasy;
```
Comment (camfiander) [inline client/src/page/loginPage.css:17]: impact font is an interesting choice :) but we can figure out what fonts we'll use globally on styling tickets

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "on-call-coordinator",
```
Comment (camfiander) [inline package-lock.json:2]: is this file supposed to be here?

File: server/routes/users.js [hunk 3]
```diff
@@ -1,15 +1,58 @@
...
+
+  const user = await getUserByEmail(req.body.email, req.body.password);
+  if (user==null) {
+
+    res.status(401);
+    res.send("User does not exist!");
+    
+  }
+
+  else {
```
Comment (camfiander) [inline server/routes/users.js:51]: Indentation here could be fixed using Format Document (`Shift-Alt-F` in VSCode or via Command Pallete)

File: server/routes/users.js [hunk 4]
```diff
@@ -1,15 +1,58 @@
...
+  return result;
+}
+ 
+
+router.post('/login', async function (req, res, next) {
+
+  const user = await getUserByEmail(req.body.email, req.body.password);
+  if (user==null) {
+
+    res.status(401);
```
Comment (camfiander) [inline server/routes/users.js:46]: good job setting the HTTP status code correctly, I forget this step often :)

File: server/routes/users.js [hunk 5]
```diff
@@ -1,15 +1,58 @@
...
+async function tokenify (user) {
+
+  
+  let jwtSecretKey = process.env.JWT_SECRET_KEY;
+  const token = jwt.sign(user, jwtSecretKey, {expiresIn: '100s'});
+  return token;
+
+}
+
+async function getUserByEmail (email, password) {
```
Comment (camfiander) [inline server/routes/users.js:27]: I can write a test for this endpoint while I'm working on fixing up testing & CI

File: server/routes/users.js [hunk 6]
```diff
@@ -1,15 +1,58 @@
...
 const prisma = new PrismaClient()
 
 /* GET users listing. */
 router.get('/', async function(req, res, next) {
   const allMyUsers = await prisma.user.findMany()
-  console.log(allMyUsers);
   res.send(allMyUsers)
 });
 
+async function tokenify (user) {
```
Comment (camfiander) [inline server/routes/users.js:18]: Not sure if this is testable right now, but we should write a test that verifies it can be decoded in a future story

**Review by camfiander — APPROVED** [review]
Left a few comments, but you should be good to merge after deleting the `package-lock.json` in the root directory

**Pull request merged**